### DISCUSSION TITLE
Error while creating a server.

### DISCUSSION THREAD

**Discussion by sabah215** [nodejs/help#5034] [Q&A] [answered]
I have written a simple code to create server on Nodejs. The code is as follows

const http = require('http');

const server = http.createServer((req, res) => {
       res.write('Welcome to our homepage');
       res.end();
});

server.listen(5000);

For this, I am getting an error saying:

<img width="839" alt="Screen Shot 2023-01-23 at 7 46 55 PM" src="https://user-images.githubusercontent.com/6989419/214188944-a915e97b-96bf-4e61-b502-21b4dd5f6c08.png">

Can Anyone please help with this

**Comment by AfrazAslam** [comment, marked as answer]
There is another program using port 5000.
Using the lsof command you can find the process ID (PID) for a specific port that is in a LISTEN state.
